pkg/framework/profile/profile.go

```diff
@@ -303,22 +303,22 @@ func NewProfile(config api.DeschedulerProfile, reg pluginregistry.Registry, opts
 	return pi, nil
 }
 
-func (d profileImpl) RunDeschedulePlugins(ctx context.Context, nodes []*v1.Node) *frameworktypes.Status {
+func (p profileImpl) RunDeschedulePlugins(ctx context.Context, nodes []*v1.Node) *frameworktypes.Status {
```

Comment:
this adds a lot of line diffs for a no-op variable rename. I'm assuming that you chose `p` here to match [`p.registryToExtensionPoints()`](https://github.com/kubernetes-sigs/descheduler/blob/b95380641fcdb31dee6026d7e9e96f226dbbd424/pkg/framework/profile/profile.go#L191)... what if we switch that function to use `d` instead, since it involves fewer changes and keeps the `git blame` history cleaner for these logic functions

Reply:
This is indeed reasonable, I will modify it

Reply:
I initially made the change because having different receiver names violates the linting style. If we don't want the code to change, I can also revert the other similar modifications of this nature ? WDYT @damemi